main.py

```diff
@@ -57,13 +57,10 @@ async def scheduler():
         await aioschedule.run_pending()
         await asyncio.sleep(1)
 
-@asyncio.coroutine
-def starter():
-    executor = ProcessPoolExecutor()
-    out = yield from loop.run_in_executor(executor, scheduler) 
-    
+def starter(): return scheduler(); #idk how multiprocessing interacts with asyncio, so...
```

Comment:
Are you sure that we need to return scheduler function? Maybe we just need to call it

Reply:
idk, that's why I left a comment on the code

Reply:
ok, have you tested it? is it works normally?